models/models.topics.js

```diff
@@ -3,7 +3,13 @@ const db = require("../db/connection.js");
 exports.selectTopics = () => {
   const SQL = `SELECT * FROM topics;`;
 
-  return db.query(SQL).then((data) => {
-    return data;
+  return db.query(SQL).then(({ rows, rowCount }) => {
+    if (rowCount === 0) {
```

Comment:
The error handling here is redundant because we have already caught a bad path in app.js earlier. No further errors are possible once this endpoint has been reached. You can remove lines 7 - 12 👍